File: 04implementation_실전/럭키 스트레이트.cpp
#include <iostream>
#include <string>

using namespace std;

string solution(string str) {
	string answer = "READY";
	int tot = 0;
	int size = str.length();

	for (int i = 0; i < size / 2; ++i)
		tot += (int)(str[i] - '0');
	
	for (int i = size / 2; i < size; ++i)
		tot -= (int)(str[i] - '0');

	if (tot == 0)
		answer = "LUCKY";

	return answer;
}

int main() {
	string str;

	cin >> str;

	string ret = solution(str);
	cout << ret;
	
	return 0;
}

File: 04Implementation_실전/럭키 스트레이트.py
n=list(map(int,input()))
l=len(n)
ltot=0
rtot=0
for i in range(l//2):
    ltot+=n[i]
for j in range(l//2,l):
    rtot+=n[j]
if ltot==rtot:
    print('LUCKY')
else:
    print('READY')